infra-as-code/bicep/modules/vwanConnectivity/README.md

```diff
@@ -0,0 +1,139 @@
+# Module:  Virtual WAN
+
+This module is used to deploy the Virtual WAN network topology and its components according to the Azure Landing Zone conceptual architecture which can be found [here](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/virtual-wan-network-topology). This module draws parity with the Enterprise Scale implementation in the ARM template [here](https://github.com/Azure/Enterprise-Scale/blob/main/eslzArm/subscriptionTemplates/vwan-connectivity.json)
+
+Module deploys the following resources:
+
+- Virtual WAN
+- Public IP
```

Comment:
isnt this optional?

Reply:
> isnt this optional?

If I make Virtual WAN resource optional, it kinda defeats the purpose of having this module because this module is for creating a VWAN resource.

As for public IP being optional, I wanted to but technically I couldn't make it optionally deployed depending on parameters. Let's have a chat during our sync up